watson/watson.py

```diff
@@ -490,3 +489,48 @@ def report(self, from_, to, current=None, projects=None, tags=None,
 
         report['time'] = total.total_seconds()
         return report
+
+    def rename_project(self, old_project, new_project):
+        """Takes a project and renames it in all affected frames."""
+        if old_project not in self.projects:
+            raise click.ClickException(style(
```

Comment:
`watson.py`shouldn't really depend on `click`, only `cli.py` should.  I know, it was used already in `Watson.__init__`, but that has been bugging me for a long time and we shouldn't add new code using it in this file. Definitely not using exceptions or functions interacting with the console from it.

In this particular case, raise a `ValueError` here and catch that in `cli.rename`.

Reply:
done